src/libstd/sync/mutex.rs

```diff
@@ -393,6 +393,19 @@ impl<T: ?Sized + fmt::Debug> fmt::Debug for Mutex<T> {
     }
 }
 
+#[unstable(feature = "std_display", issue = "24372")]
```

Comment:
I wasn't sure about the stability markers but I knew it would need *something*, thanks!